projects/mercury/src/metadata/common/LinkedDataLink.js

```diff
@@ -1,28 +1,45 @@
 import React, {useContext} from 'react';
-import {Link as RouterLink} from "react-router-dom";
 import * as PropTypes from "prop-types";
-import {Link} from '@mui/material';
-import {METADATA_PATH} from "../../constants";
+import {Box, Modal, Tooltip} from '@mui/material';
+import withStyles from '@mui/styles/withStyles';
+import CloseIcon from '@mui/icons-material/Close';
+import LinkedDataEntityPage from "./LinkedDataEntityPage";
 import UserContext from '../../users/UserContext';
-
+import styles from './LinkedDataLink.styles';
 /**
- * Renders a link to the metadata editor.
- *
- * @param props
- * @constructor
+ * Renders a link to the metadata editor in a modal dialog when clicked.
  */
-const LinkedDataLink = ({uri, children}) => {
+
+const renderModal = (classes, open, handleClose, uri) => (
+    <Modal
```

Comment:
It is suggested to use Dialog rather than lower-level Modal construct (see [https://mui.com/material-ui/react-modal/#modal](https://mui.com/material-ui/react-modal/#modal)). This would also make the dialog more consistent with the existing code and you wouldn't need custom styling class.